db/schema.rb

```diff
@@ -130,6 +130,13 @@
     t.text "notes"
   end
 
+  create_table "donation_data", force: :cascade do |t|
```

Comment:
donations will be dropped and this table will be renamed, or still the same?

Reply:
Probably renamed